server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider providing a comment to explain the significance of the 'HOSTNAME' value